build_tools/buildkite/pipelines/trusted/benchmark.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Note: this runs on trusted agents, since we want the authorization checks to
+# themselves be run on trusted agents. The pipeline itself is uploaded from
+# the main repository via bootstrap-trusted.yml and it fetches the scripts only
+# from the main repository.
+agents:
+  queue: "orchestration"
+  security: "trusted"
```

Comment:
You'll need to add this to the pre-bootstrap hook on the agents. Please do that first and roll it out to the agents

Reply:
Updated the agent `buildkite-orchestration-submitted-only-1`. Is there a base image I need to update?

Reply:
Please update https://github.com/google/iree/blob/main/build_tools/buildkite/agent/hooks/pre-bootstrap-trusted-agent.py and then push it to the agent. Currently the agents are all manually set up and maintained by me